components/alert/alert.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>alert</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="/prism.css" />
+    <link rel="stylesheet" href="all.min.css" />
+    <link rel="stylesheet" href="./alert.css" />
```

Comment:
in spite of linking so many css file in head tag, you can import all your CSS files in one style.css file & link it to head tags.